elements/asq-multi-choice-q-presenter.html

```diff
@@ -577,42 +609,64 @@
             this._setOptionCorrectness(option);
           }.bind(this));
 
-          this._updateProgress(optionsDictionary, evt.total)
+          var othersDictionary = {};
+          q.data.options.forEach(function (option) {
+            if(option.otherData){
+              var keys = Object.keys(option.otherData);
+              var data = [];
+              keys.forEach(function(k){
+                data.push({'text' : k, 'num' : option.otherData[k]})
+              });
+
+              othersDictionary[option.uid] = data
+            }
+          }.bind(this));
+
+          this._updateProgress(optionsDictionary, evt.total, othersDictionary)
         }.bind(this));
       },
 
-      _updateProgress: function(options, total){
+      _updateProgress: function (options, total, others) {
         total = total || 100;
         //simultaneously update nv3d and asq-options
 
         // we will use the local ordering of options but first we need to check
-        // that they match the database 
-        var domOpts =  this.items.map(function(el){ return el.uid});
-        if(domOpts.slice().sort().join(',')!== Object.keys(options).sort().join(',')){
+        // that they match the database
+        var allUids = this.items.map(function (el) {
+          return el.uid
+        });
+        if (allUids.slice().sort().join(',') !== Object.keys(options).sort().join(',')) {
           throw Error("remote options don't match local options")
         }
 
-        var data = domOpts.map(function(key){
-          var opt = this._updateOptionProgress(key, options[key], total);
-          var pOpt = Polymer.dom(opt.root).querySelector("asq-option-presenter");
-          var txt = opt.textContent//pOpt.$.label.textContent;
-          return {
-            label: txt,
-            value: options[key] || 0
+        var data = [];
+
+        allUids.forEach(function (key) {
+          var opt = this._updateOptionProgress(key, options[key], total, others[key]);
+
+          var txt, value;
```

Comment:
add comments there about what's happening, for example:

// Read the custom text of each "other" option